selfdrive/camerad/cameras/camera_common.cc

```diff
@@ -318,6 +318,11 @@ float set_exposure_target(const CameraBuf *b, int x_start, int x_end, int x_skip
 extern ExitHandler do_exit;
 
 void *processing_thread(MultiCameraState *cameras, CameraState *cs, process_thread_cb callback) {
+  static PubMaster pm({"roadCameraState", "driverCameraState", "thumbnail"
```

Comment:
why is this static?

Reply:
processing_thread will be called from multiple camera threads.use static to make sure Pubmaster only initialized once